exercise_05/isAdditionGreaterThanFifty.js

```diff
@@ -4,3 +4,8 @@
  * input: number (summand2)
  * output: boolean
  */
+function isAdditionGreaterThanFifty(summand1, summand2) {
+    return (summand1 + summand2) >= 50;
```

Comment:
Avoid magic numbers. 

```suggestion
    return (summand1 + summand2) >= THRESHOLD;
```

- [ ] Extract number to constant